README.md

```diff
@@ -16,3 +16,27 @@ repos:
       - id: prettier-package-xml
       - id: sort-package-xml
 ```
+
+## Hooks available
+
+### `prettier-xacro-xml`
+
+Apply prettier to xacro.
+
+### `prettier-launch-xml`
+
+Apply prettier to launch.xml
+
+### `prettier-package-xml`
+
+Apply prettier to package.xml
+### `sort-package-xml`
+
+Sorts the names contained in each tag in package.xml.
+If you want to exclude a tag from the sort, add `<! -- nolint -->` at the beginning of the line.
+
+Example
```

Comment:
[IMO] This is redundant.

```suggestion
```

Reply:
Or, writing like this (`, for example:`) looks good.

https://github.com/pre-commit/pre-commit-hooks#sort-simple-yaml